api/cmd/inference-logger/main.go

```diff
@@ -38,6 +38,11 @@ var (
 	logMode          = flag.String("log-mode", string(merlinlogger.LogModeAll), "Whether to log 'request', 'response' or 'all'")
 	inferenceService = flag.String("inference-service", "my-model-1", "The InferenceService name to add as header to log events")
 	namespace        = flag.String("namespace", "my-project", "The namespace to add as header to log events")
+
```

Comment:
On this note, it seems kserve is not passing the `port` and `workers` flags. Do we need them?

Reply:
We don't need them and kserve also doesn't pass them. However, they didn't raise the error because they have default value configured. I'm keeping them for now as it would introduce more changes and we can ignore them.